tests/ImageSharp.Sandbox46/ImageSharp.Sandbox46.csproj

```diff
@@ -18,17 +18,13 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="BitMiracle.LibJpeg.NET" Version="1.4.280" />
+    <PackageReference Include="NETStandard.Library" Version="1.6.1" />
     <PackageReference Include="xunit" Version="2.3.0-beta4-build3742" />
     <PackageReference Include="Moq" Version="4.7.99" />
     <!--<PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />-->
     <!--<PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />-->
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="..\ImageSharp.Tests\**\*.cs" Exclude="bin\**;obj\**" Link="Tests\%(RecursiveDir)%(Filename)%(Extension)" />
```

Comment:
Why have we touched this? Are we now able to do the profiling we needed this for. @antonfirsov What are your thoughts?

Reply:
I think the reason might be similar to the `.vscode/tasks.json` removal.

@caetanator can we ask you to undo all changes in this PR which are workarounds specific to your work environment?

Reply:
This project doesn’t compile it says the namespace "Tests" doesn’t exist.

I tried some suggestions from VS2016 to fix it, but none of them worked.

I think that one of this fixes made the change.

All changes outside the BMP file format, project configurations, and documentation can be ignored, as they were made for testing proposes or allowing your code to compile on my platform.

Please feel free to do any changes you like to de code.